docs/guides/latest/backends/local.md

```diff
@@ -129,6 +131,8 @@ $ kubectl apply -f https://github.com/stashed/docs/raw/{{< param "info.version"
 repository/local-repo-with-nfs created
 ```
 
+>For NFS backend, Stash may have to run the network volume accessor deployments in privileged mode to provide Snapshot listing facility. In this case, please configure network volume accessors by following the instruction [here](/docs/setup/install.md#configuring-network-volume-accessor).
+
```

Comment:
`(/docs/setup/install.md#configuring-network-volume-accessor)` is not a valid link